src/main/flight/pid.c

```diff
@@ -594,6 +621,43 @@ static void FAST_CODE applyItermRelax(const int axis, const float gyroRate, floa
         }
     }
 }
+#ifdef USE_D_BOOST
+static float FAST_CODE applyDBoost(float rateTarget, float gyroRate, flight_dynamics_index_t axis) {
+    
+    float dBoost = 1.0f;
+    
+    if (dBoostFactor > 1) {
+        const float dBoostGyroDelta = (gyroRate - previousRateGyro[axis]) / dT;
+        const float dBoostGyroAcceleration = fabsf(biquadFilterApply(&dBoostGyroLpf[axis], dBoostGyroDelta));
+        const float dBoostStickAcceleration = fabsf((rateTarget - previousRateTarget[axis]) / dT);
```

Comment:
`dBoostStickAcceleration` is probably not the best name here, we're examining `rateTarget` which might be set by `LEVEL` mode as well.